custome/mycart.css

```diff
@@ -0,0 +1,101 @@
+.my-cart {
+    position: relative;
+    top: 3rem;
+    text-align: center;
+    margin: 1rem;
+}
+
+.cart-main {
+    display: flex;
```

Comment:
for Flex and its properties you can create a common helper class and use it everywhere. Just a suggestion